skyportal/handlers/api/user.py

```diff
@@ -19,50 +20,119 @@
     Stream,
 )
 
+from skyportal.model_util import role_acls, all_acl_ids
 
+log = make_log("api/user")
 env, cfg = load_env()
 
 
+def set_default_role(user, session):
```

Comment:
Can we have a few short doc strings? I think it might be worth calling out that you need to commit outside of the function.

Reply:
Sure thing ! Will add that later today